twitchpopups.htm

```diff
@@ -91,15 +117,15 @@ <h1 id="popuptext"></h1>
                   hotseatison = true;
                   $("#popupbox").show();
                   hotseatuser = commandName.substr(10).toLowerCase();
-                  $("#popupbox").css({"background-color":"#aa1100"}); 
-                  $("#popuptext").text("🔥 " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT 🔥");
+                  $("#popupbox").css({"background-color":hotseatBg}); 
+                  $("#popuptext").text(hotseatEmoji + " " + hotseatuser.toUpperCase() + " IS IN THE HOTSEAT " + hotseatEmoji);
                   doAnimation();
                 }
             }
                 
             if (hotseatison) {
                 if (context.username == hotseatuser){

```

Comment:
match the type too `===`

Reply:
Not a part of these commits. Please feel free to fork the repo and fix it.